### DISCUSSION TITLE
添加文本对比功能

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#183] [Ideas]
希望添加文本对比功能，在需要查看两段相似文本是否相同的时候十分有效。有其实对于代码， json，yaml 等格式的文本。

相似网站： [textDifference](https://www.jq22.com/textDifference)

**Comment by lyzhang1999** [comment]
你好，He3 包含文本对比工具：https://t.he3app.com?aake  想问一下是没有搜索到这款工具吗？

**Reply by alctny** [reply]
最开始是通过 “对比”，"compare", "com" 等关键字进行查询，确实没有查询到相应结果。然后挨个功能查找，也没有找到。看到你的回复之后又找了一遍，确实找到了处于 “文本” 组下的 “文本比较” 功能。

但是，我再次进行搜索，结果是：
搜索 “文本” --> 包含 “文本对比”
搜索 “对比” / "comp“ / "diff"  --> 没有结果 --> 试用语义搜索 --> 包含 yaml，json，图片对比，并不包含"文本对比"

**Reply by alctny** [reply]
如果是出于匹配到的对象太多，不方便全部做展示。那么我个人认为，这个地方，也应该存在让用户看到所有结果的选项。（事实上，上面几乎每一个关键词都没有能匹配到所有结果）

**Reply by lyzhang1999** [reply]
感谢您的建议，搜索这块目前正在进行重构，将进行重新优化。

**Comment by hongchaodeng** [comment]
@alctny 非常感谢你的反馈！

由于当前版本对搜索条件限制得太死，导致很多结果搜不出来。这个在下一版将有显著改善。敬请期待！

<img width="690" alt="Snipaste_2023-05-19_12-26-33" src="https://github.com/he3-app/he3-feedback/assets/920884/6dbd97d9-44df-4604-93b9-9ae477363563">

<img width="606" alt="Snipaste_2023-05-19_12-26-59" src="https://github.com/he3-app/he3-feedback/assets/920884/93fa9344-ca95-4a2d-b9cf-50e534135405">
